AzureKeyVaultUtils/Helpers/KeyVaultHelper.cs

```diff
@@ -0,0 +1,84 @@
+﻿using Azure.Identity;
+using Azure.Security.KeyVault.Secrets;
+using Azure;
+using System.Text.Json;
+
+namespace AzureKeyVaultUtils.Helpers
+{
+    public class KeyVaultHelper
+    {
+        private readonly SecretClient _client;
+
+        public KeyVaultHelper(string keyVaultUrl)
+        {
+            _client = new SecretClient(new Uri(keyVaultUrl), new InteractiveBrowserCredential());
+        }
+
+        /// <summary>
+        /// An asynchronous task that recursively inserts data into Azure Key Vault by iterating over the properties of a JsonElement data.
+        /// </summary>
+        /// <param name="data"></param>
+        /// <param name="prefix"></param>
+        /// <returns></returns>
+        public async Task Insert(JsonElement data, string prefix = "")
```

Comment:
This method shouldn't be part of this class. KeyVaultHelper class - JsonElement as input doesn't make sense.